libpod/kube.go

```diff
@@ -452,13 +451,7 @@ func newServicePortState() servicePortState {
 }
 
 func truncateKubeAnnotation(str string, useLongAnnotations bool) string {
-	str = strings.TrimSpace(str)
-	if useLongAnnotations || utf8.RuneCountInString(str) < define.MaxKubeAnnotation {
-		return str
-	}
-	trunc := string([]rune(str)[:define.MaxKubeAnnotation])
-	logrus.Warnf("Truncation Annotation: %q to %q: Kubernetes only allows %d characters", str, trunc, define.MaxKubeAnnotation)
-	return trunc
+	return str
```

Comment:
Shouldn't this function be removed?

Reply:
That’s what I would do. Just that I was not sure whether the maintainers want the same. I will remove it.